app/routes/wallet/dids.py

```diff
@@ -99,28 +147,57 @@ async def set_public_did(
     return result
 
 
-@router.patch("/{did}/rotate-keypair", status_code=204)
+@router.patch("/{did}/rotate-keypair", status_code=204, summary="Rotate Key Pair")
 async def rotate_keypair(
     did: str,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> None:
+    """
+    Rotate key pair for DID
+    ---
+
+    This endpoint allows you to rotate the key pair for a DID.
+
+    Parameters:
+    ---
+        did: str
+
+    Response:
+    ---
+        204 No Content
+
+    """
     bound_logger = logger.bind(body={"did": did})
-    bound_logger.info("PATCH request received: Rotate keypair for DID")
+    bound_logger.info("PATCH request received: Rotate key pair for DID")
     async with client_from_auth(auth) as aries_controller:
-        bound_logger.debug("Rotating keypair")
+        bound_logger.debug("Rotating key pair")
         await handle_acapy_call(
             logger=logger, acapy_call=aries_controller.wallet.rotate_keypair, did=did
         )
 
-    bound_logger.info("Successfully rotated keypair.")
+    bound_logger.info("Successfully rotated key pair.")
```

Comment:
Ensure that rebasing includes all these info logs have been moved to debug level